modules/video/template.html

```diff
@@ -0,0 +1,3 @@
+<section class="snowball-block snowball-block-video {{full-width}}">
+  <iframe style="{{custom-style}}" width="560" height="315" src="https://www.youtube.com/embed/{{video-id}}?autoplay=0&showinfo=0&controls=0&rel=0&iv_load_policy=3&disablekb=0" frameborder="0"></iframe>
```

Comment:
Is there a reason the width is set to 560 instead of 600?

Also, how about we set width and height of iframe as a scoped style instead of inline attributes?